src/TSID/include/BipedalLocomotion/TSID/FeasibleContactWrenchTask.h

```diff
@@ -98,14 +95,10 @@ class FeasibleContactWrenchTask : public TSIDLinearTask
     bool update() override;
 
     /**
-     * Enable the task. I.e. a contact wrench different from zero is allowed.
+     * Set if the contact is active or not.
+     * @param isActive true if the contact is active, false otherwise.
      */
-    void enable();
-
-    /**
-     * Disable the task. I.e. the contact wrench is forced to be equal to zero.
-     */
-    void disable();
+    void isContactActive(bool isActive);
```

Comment:
I realized later that ``isContactActive`` is more in the form of a question, so I would expect a return value. Maybe ``setContactActive`` would suggest more that you are changing an internal state. Does it make sense?

Reply:
Fixed in https://github.com/dic-iit/bipedal-locomotion-framework/pull/369/commits/1277a6264d7dc26fc4812a7152ac8b071905bbab